### ISSUE TITLE
Unable to parse function metadata with a colon in the attribute name

### ISSUE THREAD

**Issue by atuttle** [cflint/CFLint#102]
As of CF11, this syntax is supported:

    function setStatus( rc ) my:role="support:admin" {}

... with the relevant bit being: `my:role="..."`

Attempting to lint this returns:

```
cflint -file tickets.cfc
line 45:28 mismatched input ':' expecting '='
null
<issues>
  <issue severity="ERROR" id="PARSE_ERROR" message="PARSE_ERROR" category="CFLint" abbrev="PE">
    <location file="/Users/adam/dev/{snip}/tickets.cfc" fileName="tickets.cfc" column="28" line="45" message="Unable to parse" variable="">
      <Expression><![CDATA[[@284,1213:1213=':',<62>,45:28]]]></Expression>
    </location>
  </issue>
</issues>
```

**Comment by ryaneberly** [comment]
@atuttle thanks for reporting this.
It fixed in cfparser/cfparser master.  If you want to use it before it is released, you need to build cfparser and CFLint yourself.

In the future, it's easier for us if you report parsing errors directly on the cfparser project.  Thanks!

**Comment by ryaneberly** [comment]
fixed in 5.1-snapshot
